src/connection.cpp

```diff
@@ -140,19 +140,19 @@ namespace oxen::quic
         return 0;
     }
 
-    int recv_rx_key(ngtcp2_conn* conn, ngtcp2_encryption_level level, void* user_data)
+    int recv_rx_key(ngtcp2_conn* /*conn*/, ngtcp2_encryption_level /*level*/, void* /*user_data*/)
     {
         // fix this
         return 0;
     }
 
-    int recv_tx_key(ngtcp2_conn* conn, ngtcp2_encryption_level level, void* user_data)
+    int recv_tx_key(ngtcp2_conn* /*conn*/, ngtcp2_encryption_level /*level*/, void* /*user_data*/)
```

Comment:
function itself currently unused, behavior currently facilitated by the ngtcp2-via-gnutls implementation

Reply:
@tewinget will this be needed for the null crypto implementation you're working on? (same question for your comment on `recv_rx_key` below)

Reply:
Something to this effect will be, yes.